Emulator Support and Issues overhaul

Nepiki opened the pull request:
Big overhaul of the Emulator Support and Issues docs, which includes:

- Converting all lists to tables.
- Adding previously-missed hardcore-compliant emulators (DolphinUWP, SkyEmu, Yaba Sanshiro).
- Remove Delta from each system.
- Add intro with hardcore-compliant frontends (including Delta).
- Re-arrange Sega systems to follow on-site naming.

Nepiki committed:
Emulator Support and Issues overhaul

Big overhaul of the Emulator Support and Issues docs, which includes:

- Converting all lists to tables.
- Adding previously-missed hardcore-compliant emulators (DolphinUWP, SkyEmu, Yaba Sanshiro).
- Remove Delta from each system.
- Add intro with hardcore-compliant frontends (including Delta).
- Re-arrange Sega systems to follow on-site naming.

Nepiki committed:
Update emulator-support-and-issues.md

Add the following:

- NES RA Adapter for NES, type: Console adapter
- RA2Snes for SNES, type: Console adapter

wescopeland commented on docs/general/emulator-support-and-issues.md:
```diff
@@ -9,341 +9,428 @@ description: Comprehensive guide to hardcore compliance and compatibility for Re
 This page focuses on hardcore-compliant emulators. For detailed compliance requirements, see the [Hardcore Compliance Requirements](/general/hardcore-compliance-requirements) page. For extensive notes on unsupported emulators and cores, check the [unsupported emulators page](/developer-docs/unsupported-emulators-and-cores).
 :::
 
+libretro cores can be played on the following hardcore-compliant frontends. You can find each of them on the [Downloads page](https://retroachievements.org/downloads), listing all cores available for them.:
```
```suggestion
libretro cores can be played on the following hardcore-compliant frontends. You can find each of them on the [Downloads page](https://retroachievements.org/downloads), listing all cores available for them:
```

wescopeland commented on docs/general/emulator-support-and-issues.md:
```diff
@@ -9,341 +9,428 @@ description: Comprehensive guide to hardcore compliance and compatibility for Re
+| **FCEUmm**                                                     | libretro core       | Most recommended. |
+| **Mesen**                                                      | libretro core       |                   |
+| **[RANes](https://retroachievements.org/downloads)**           | Standalone emulator |                   |
+| **[NES RA Adapter](https://github.com/odelot/nes-ra-adapter)** | Console adapter     |                   |
```
I believe this should be placed under NES/Famicom, not NES/Famicom Disk System.

wescopeland commented on docs/general/emulator-support-and-issues.md:
```diff
@@ -9,341 +9,428 @@ description: Comprehensive guide to hardcore compliance and compatibility for Re
+- [RetroArch](https://www.retroarch.com/index.php?page=platforms) - Supports 45 systems.
+- [RALibRetro](https://retroachievements.org/downloads) - Supports 45 systems.
+- [Firelight](https://biscuitcakes.itch.io/firelight) - Supports 10 systems.
+- [Delta (iOS)](https://apps.apple.com/us/app/delta-game-emulator/id1048524688) - Supports 7 systems.
```
These hardcoded numbers (45 / 45 / 10 / 7) are likely to go out of date quickly. Approximation might be better, ie:
40+ / 40+ / 10+ / 5+

wescopeland commented on docs/general/emulator-support-and-issues.md:
```diff
@@ -9,341 +9,428 @@ description: Comprehensive guide to hardcore compliance and compatibility for Re
+| **Mupen64Plus-Next**                                                            | libretro core       | - Most recommended.<br>- Separated into cores for OpenGL ES 2 and 3. |
+| **ParaLLEl N64**                                                                | libretro core       |                                                                      |
+| **[RAProject64](https://retroachievements.org/downloads)**                      | Standalone emulator |                                                                      |
+| **[Luna Project64](https://github.com/Luna-Project64/Luna-Project64/releases)** | Standalone emulator |                                                                      |
```
Typo (carried over from the old document): this is actually called "Luna's Project64".

wescopeland commented on docs/general/emulator-support-and-issues.md:
```diff
@@ -9,341 +9,428 @@ description: Comprehensive guide to hardcore compliance and compatibility for Re
+
+| Name          | Type          | Notes                                                                                                                                                     |
+| :------------ | :------------ | :-------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **PicoDrive** | libretro core | - Several games are problematic, use BizHawk if an achievement shows as unsupported or the game performs poorly.<br>- Appears to still have unmapped RAM. |
+| **PicoDrive** | BizHawk core  | Most recommended.                                                                                                                                         |
```
Recommend swapping the order of these two rows.

Nepiki replied:
The reasoning behind the ordering is that, for every table, I went libretro core => BizHawk core => Standalone emulator => Console adapter. This is purely a me thing though, so if you feel it's better to change order I have no objections :)

wescopeland commented on docs/general/emulator-support-and-issues.md:
```diff
@@ -9,341 +9,428 @@ description: Comprehensive guide to hardcore compliance and compatibility for Re
+| Name                                                                    | Type                | Notes                                                                                                            |
+| :---------------------------------------------------------------------- | :------------------ | :--------------------------------------------------------------------------------------------------------------- |
+| **[Dolphin](https://retroachievements.org/download.php#dolphin)**       | Standalone emulator | - Must use version 2407-68 or newer.<br>- Ensure "Enable Dual Core (speedup)" is unchecked as it is unsupported. |
+| **[DolphinUWP](https://github.com/SternXD/dolphin/releases/tag/1.1.9.0) | Standalone emulator |                                                                                                                  |
```
Missing a closing `**` here:
<img width="234" height="67" alt="Screenshot 2025-08-11 at 7 59 03 PM" src="https://github.com/user-attachments/assets/811b80df-99bb-47f6-b2f7-b04bbcdb0965" />

wescopeland commented on docs/general/emulator-support-and-issues.md:
```diff
@@ -9,341 +9,428 @@ description: Comprehensive guide to hardcore compliance and compatibility for Re
-
-### Sega Saturn
+::: info
+Appear to still have unmapped RAM.
```
```suggestion
These cores appear to still have unmapped RAM.
```

wescopeland commented on docs/general/emulator-support-and-issues.md:
```diff
@@ -9,341 +9,428 @@ description: Comprehensive guide to hardcore compliance and compatibility for Re
 
-- ✅ libretro core: **FreeChaF**
+| Name                                                  | Type                | Notes |
+| :---------------------------------------------------- | :------------------ | :---  |
```
```suggestion
| :---------------------------------------------------- | :------------------ | :---- |
```

wescopeland reviewed (commented):
Looks good - found a few odds and ends.

Nepiki committed:
Update docs/general/emulator-support-and-issues.md

Co-authored-by: Wes Copeland <wlcopeland1@gmail.com>

Nepiki committed:
Update docs/general/emulator-support-and-issues.md

Co-authored-by: Wes Copeland <wlcopeland1@gmail.com>

Nepiki committed:
Update docs/general/emulator-support-and-issues.md

Co-authored-by: Wes Copeland <wlcopeland1@gmail.com>

Nepiki committed:
Update emulator-support-and-issues.md

Applied feedback by wescopeland, updating the following:

- Move RA NES Adapter to NES/Famicom rather than disk system.
- Change supported systems by frontends to approximation rather than hardcoded numbers.
- Correct naming for Luna's Project64.
- Fix bold usage for DolphinUWP.

wescopeland reviewed (approved):
Thanks!

CasualPokePlayer commented on docs/general/emulator-support-and-issues.md:
```diff
@@ -9,341 +9,428 @@ description: Comprehensive guide to hardcore compliance and compatibility for Re
+- [RetroArch](https://www.retroarch.com/index.php?page=platforms) - Supports 40+ systems.
+- [RALibRetro](https://retroachievements.org/downloads) - Supports 40+ systems.
+- [Firelight](https://biscuitcakes.itch.io/firelight) - Supports 10+ systems.
+- [Delta (iOS)](https://apps.apple.com/us/app/delta-game-emulator/id1048524688) - Supports 5+ systems.
```
Delta does not use libretro, it uses its own core system (like BizHawk does).
